mne/viz/backends/_pysurfer_mayavi.py

```diff
@@ -179,22 +179,31 @@ def tube(self, origin, destination, radius=0.001, color='white',
         origin = np.atleast_2d(origin)
         destination = np.atleast_2d(destination)
         if scalars is None:
-            surface = self.mlab.plot3d([origin[:, 0], destination[:, 0]],
-                                       [origin[:, 1], destination[:, 1]],
-                                       [origin[:, 2], destination[:, 2]],
-                                       tube_radius=radius,
-                                       color=color,
-                                       figure=self.fig)
+            for idx in range(origin.shape[0]):
+                surface = self.mlab.plot3d([origin[idx, 0],
+                                            destination[idx, 0]],
+                                           [origin[idx, 1],
+                                            destination[idx, 1]],
+                                           [origin[idx, 2],
+                                            destination[idx, 2]],
+                                           tube_radius=radius,
+                                           color=color,
+                                           figure=self.fig)
```

Comment:
This could potentially be much slower if there is a long tube (adding thousands or tens of thousands of single tubes will probably have much worse drawing overhead versus a single contiguous tube).

- Maybe there should be a `mode='continuous'` vs `mode='separate'` or something?
- Alternatively, we could support different modes implicitly depending on whether you get an array of shape `(n_tubes, n_segments, 3)` (`n_tubes` contiguous tubes, each separate from one another) or just `(n_segments, 3)` (single contiguous tube).
- The best solution would probably be to figure out why PyVista and VTK are different. @GuillaumeFavelier at some point it would be good to look deeper into how PyVista and Mayavi do these differently at the VTK level. It seems worth it in the long run to be able to change these properties without having to create a ton of objects. 

For now @rob-luke can you just add a comment listing this as a TODO for when we run into a performance problem with this in the future?

Reply:
Added a TODO and link to issue that @GuillaumeFavelier raised